daprdocs/content/en/reference/components-reference/supported-secret-stores/file-secret-store.md

```diff
@@ -54,15 +57,32 @@ Given the following json:
 }
 ```
 
-The store will load the file and create a map with the following key value pairs:
+If `multiValued` is `"false"`, the store will load the file and create a map with the following key value pairs:
 
 | flattened key           | value                           |
 | ---                     | ---                             |
 |"redis"                  | "your redis password"           |
 |"connectionStrings:sql"  | "your sql connection string"    |
 |"connectionStrings:mysql"| "your mysql connection string"  |
 
-Use the flattened key (`connectionStrings:sql`) to access the secret.
+Use the flattened key (`connectionStrings:sql`) to access the secret. The following JSON map returned:
+
+```json
+{
+  "connectionStrings:sql": "your sql connection string"
+}
+```
+
+If `multiValued` is `"true"`, you would instead use the top level key. In this example, `connectionStrings` would return the following map:
```

Comment:
Please, also mention how the subtrees are flat here as well and appended with the separator. It was the main difference between your approach and the one I had in mind. I actually think your approach is better and should be documented here for user's benefit.